### PR TITLE
RELATED: RAIL-3419 Update dependencies

### PR DESCRIPTION
**Pull request by no23reason**
The following dependencies were updated:

* codemirror
* elliptic
* hosted-git-info
* http-proxy
* ini
* react-scripts
* serve
* ssri
* ua-parser-js
* websocket-extensions
* y18n
* yup

JIRA: RAIL-3419

### CODE REVIEW

**Comment by yenkins** [comment]
## The code coverage is **the same** after your change 🤟
This is a good thing if you are not sure ;)
#### Report (compared to build [#224](https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/224/))
Covered lines: **23.39%**, change: **0%**
Covered branches: **38.46%**, change: **0%**
#### Coverage report
https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/225/artifact/ci/results/coverage/lcov-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/225/ : SUCCESS in 1m 14s
- https://checklist.intgdc.com/job/compliance/job/gooddata-create-gooddata-react-app-oss-check/188/ : SUCCESS in 7s

File: bootstrap/tsconfig.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
         "suppressImplicitAnyIndexErrors": true,
         "useDefineForClassFields": true,
         "strict": true,
+        "noFallthroughCasesInSwitch": true
```
Comment (no23reason) [inline bootstrap/tsconfig.json:24]: This was added by running the `yarn start` with the new react-scripts version.

**Review by kandl — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
## The code coverage is **the same** after your change 🤟
This is a good thing if you are not sure ;)
#### Report (compared to build [#224](https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/224/))
Covered lines: **23.39%**, change: **0%**
Covered branches: **38.46%**, change: **0%**
#### Coverage report
https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/226/artifact/ci/results/coverage/lcov-report/index.html

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-create-gooddata-react-app-merge-compliance-check/82/ : SUCCESS in 19s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/226/ : SUCCESS in 56s
- https://checklist.intgdc.com/job/compliance/job/gooddata-create-gooddata-react-app-fossa-scanning-zuul-el7/47/ : FAILURE in 4m 03s (non-voting)

**Pull request merged**